ramls/holdingsrecordssource.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A holdings records source",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The unique ID of the holdings records source; UUID",
+      "$ref": "uuid.json"
+    },
+    "name": {
+      "type": "string",
+      "description": "Name of the holdings records source"
+    },
+    "source": {
+      "type": "string",
+      "description": "The holdings records source"
+    }
```

Comment:
Scenarios 6 and 7 of [UIIN-1229](https://issues.folio.org/browse/UIIN-1229) state that the two options for this property are `folio` and `local`.

I think we should consider defining an enum for this property to disallow any other values.

Reply:
Is that compatible with Story point 4?

https://issues.folio.org/browse/MODINVSTOR-542

"
    When the holdings record is being maintained with an underlying MARC record in the MFHD format
    Then the value of the source element is MARC
"

Reply:
Yes, as I understand it. I believe the `source element` in that scenario is the `source` for the holdings record, rather than the `source` for the holdings source itself. Which I fully admit is really confusing and frustrating :-/